index.js

```diff
@@ -127,7 +127,12 @@ function morgan (format, options) {
       }
 
       debug('log request')
-      stream.write(line + '\n')
+      if (stream.writableObjectMode && typeof line == 'object') {
```

Comment:
I know you made this a while ago, but I was just pointed to it, haha.

So I love this idea! I'm curious if there is the `typeof line == 'object'` is really needed at all or not -- for example if the `line` is a string, perhaps just give that as-is as well to the object-mode stream without adding the trailing newline. I could see it being useful for the stream to concatenate with a different character, for example.

Reply:
The only reason I added `typeof line == 'object'` was to minimize the changes to existing behavior.  As the library stands right now, if you were to log a string, it would add a newline to that string regardless of whether your stream supported object mode or not.  That type check was to make sure this patch would not alter that behavior.

I like your design (without the type check) better as a design, but I thought the patch would be more likely to be accepted if it did not change any existing behavior.